src/HIE/Bios/Ghc/Api.hs

```diff
@@ -83,13 +83,13 @@ initializeFlagsWithCradleWithMessage msg fp cradle =
 initSessionWithMessage :: (GhcMonad m)
             => Maybe G.Messager
             -> ComponentOptions
-            -> m ()
+            -> m G.SuccessFlag
```

Comment:
If we are using types from ghc, we should add it to the list of exported symbols. Also `succeeded` and `failed` should be added to the API module export. Otherwise the call-site must add an explicit dep to ghc.